pkg/kdump/kdump.sh

```diff
@@ -39,18 +34,13 @@ if test -f /proc/vmcore; then
 
     TS=$(date -Is)
 
-    # Collect a minimal kernel dump for security reasons
-    KDUMP_PATH="$DIR/kdump-$TS.dump"
-    makedumpfile -d 31 /proc/vmcore "$KDUMP_PATH" > /dev/null 2>&1
```

Comment:
what if we compress the dump file with `-c`?

Reply:
To be honest, it was never needed to go such deep and debug kernel using dumps. We collect full dmesgs, which is almost always enough (kdumps were introduced by me as a mechanism for debugging when I just joined the company and it was not quite clear what we do actually, so I had an impression this is what we really need, which apparently is not). As I understood, that space in /persist is quite precious, so here space wins and dumps lose.